### PR TITLE
Docs: Add use case for find command

### PR DESCRIPTION
**Pull request by RyanNgWH**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -654,11 +654,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Use case**: UC05 - Find a client
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to find a client by an attribute.
+1. Fitbook displays a success message with the number of clients found, along with the filtered list of clients.
+1. Use case Ends
```
Comment (dillontkh) [inline docs/DeveloperGuide.md:672]: Should we standardise having a full stop at the end of every sentence?

**Review by dillontkh — APPROVED** [review]
LGTM apart from the above comment!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (RyanNgWH) [reply docs/DeveloperGuide.md:672]: Oh good catch! Fixed it!

**Review by ruijietay — APPROVED** [review]
LGTM

**Review by bgopi23 — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -654,11 +654,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 * 2a. User enters an invalid command
-    * 2b1. FitBook alerts the user that the command is invalid and displays the correct format
+    * 2a1. FitBook alerts the user that the command is invalid and displays the correct format
     * Use case resumes
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Find a client
```
Comment (jjchee77) [inline docs/DeveloperGuide.md:664]: Should use plural

**Review by jjchee77 — APPROVED** [review]
LGTM! Just one nit to pick

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jjchee77) [reply docs/DeveloperGuide.md:664]: And adjust accordingly for the use case in general.

**Pull request merged**